src/bufr/IodaEncoder/IodaDescription.cpp

```diff
@@ -89,4 +111,20 @@ namespace Ingester
     {
         variables_.push_back(variable);
     }
+
+    void IodaDescription::setBackend(std::string backend)
+    {
+        if (backend == "netcdf")
```

Comment:
if the code supports "inmemory" and "in-memory", can we throw in here "netCDF" and "NetCDF" just for safety's sake?

Reply:
Maybe any case variation of netCDF?

Reply:
Backend is now case insensitive.